.gitignore

```diff
@@ -1 +1,2 @@
 /vendor/
+.idea/
```

Comment:
By the way, rather than put IDE-specific ignores in the repository's .gitignore (some of us don't use Goland), you can put this directive in`~/.gitignore_global`.

Reply:
Thanks for the tip :) It would be nice to tag a new release so that people can cleanup their dependencies :)